### ISSUE TITLE
Outline Review

### ISSUE THREAD

**Issue by adelnehme** [datacamp/Machine-Learning-With-XGboost-live-training#1]
Hi @lisdc 👋 

Specs are super-detailed and clean - thanks for the effort put into them! Here are my comments:

### What problem(s) will students learn how to solve? (minimum of 5 problems)

:star:

### What technologies, packages, or functions will students use? Please be exhaustive.

:star:

### What terms or jargon will you define?

- [ ] 🔍  **1.** I think what sets apart these excellent sessions from good ones is the ability to explain things rather intuitively - I'd love to see some use of analogies/gifs/images to clarify these terms and jargons as you go along the session 😄 

### What mistakes or misconceptions do you expect?

:star:

### What level of expertise should learners have before beginning the live training?

- [ ] 🤔 **2.** Albeit I agree with this "Can define hyperparameter tuning, underfitting, overfitting, the bias-variance tradeoff, regularization, classification, regression, cross-validation, grid search, and random search" - it's good to also remind students of these different concepts so we keep it accessible. 

## Step 3: Prerequisites

- [x] 🤔 **3.** Not sure if we should only keep "Extreme Gradient Boosting with XGBoost" since chapter 1 kinda covers decision trees and walks them through xgboost - but I think an upside is that it also sends a message that this is a relatively advanced session. 

## Step 4: Outline

- [ ] 🤔  **4.** I'm not sure if we should dig into the different types of weak learners in the introduction slides - since we can explain this during the session on markdown as we go about modeling and then you have more leeway to discuss when to use a specific type of learner given the problem/dataset. What do you think?

- [x] 🔧  **5.** Recommend adding a 5-minute section before "Your first XGBoost Model" dedicated to introducing the dataset, the fact it has already been pre-processed, and potentially list a few of the steps applied to the dataset. This puts your session into perspective and situates in the machine learning pipeline. 

- [x] 🤔  **6.** I think the **_"Digging into Parameters"_** section could be the star of the show if students come out understanding the intuition behind some of the hyperparameters you plan on covering and how changing them affects model performance. 

- [x] 🤔 **7.** We've already briefly touched upon this - but consider performance bottlenecks when doing hyperparameter tuning and using imported models to avoid those bottlenecks.
